easybuild/easyconfigs/m/miniprot/miniprot-0.12-GCC-12.2.0.eb

```
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp'
+
+name = 'miniprot'
+version = '0.12'
+
+homepage = 'https://github.com/lh3/miniprot'
+description = """Miniprot aligns a protein sequence against a genome with affine gap penalty, splicing and frameshift.
+It is primarily intended for annotating protein-coding genes in a new species using known genes from other species."""
+
+toolchain = {'name': 'GCC', 'version': '12.2.0'}
+
+github_account = 'lh3'
+
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['869df52d8a3d350e0a5ba6eb0d1685e78bb7087ace9870d196f5fb4761c3a692']
+
+dependencies = [
+    ('zlib', '1.2.12'),
+]
+
+files_to_copy = [(['%(name)s'], 'bin')]
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s'],
+    'dirs': [],
+}
+
+sanity_check_commands = [('%(name)s || test $? -gt 0')]
```

```suggestion
sanity_check_commands = ['%(name)s 2>&1 | grep "Usage: %(name)s"']
```

This way you are testing the commad or any exit code > 0 which means the command failed.d
Since `minprot -h` returns 1 and prints the help to `stderr` we can just check if the help message is present or not